clients-simple/src/main/java/uk/gov/gchq/palisade/clients/simpleclient/config/ClientConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.clients.simpleclient.config;
+
+import uk.gov.gchq.palisade.Generated;
+
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class ClientConfiguration {
```

Comment:
What is the purpose of this class?

Reply:
its used by the UrlDataClient to create a map for the feign builder, i.e <data-service, localhost:808*> 
It was a part of ApplicatonConfiguration and then Autowired in but this was a bit hacky so a bean was needed

Reply:
Another benefit is that the data-services don't all need to be specified in advance - when using eureka or k8s, there's a good chance we can just use the connection detail as-is, so we can look up the name in this map (web.clients) and then have some default behaviour if the key is missing (use the key as the value) (still need to specify every possible data-service for static, but that's to be expected)